homeassistant/components/media_player/androidtv.py

```diff
@@ -0,0 +1,413 @@
+"""
+Provide functionality to interact with AndroidT devices on the network.
+Example config:
+
+media_player:
+  - platform: androidtv
+    devices:
+      192.168.1.37:
+        name: MIBOX3 ANDROID TV
+        port: 5555
+        scan_interval: 10
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/media_player.androidtv/
+"""
+
+import logging
+from datetime import timedelta
+import voluptuous as vol
+
+from homeassistant.components.media_player import (
+    DOMAIN, MediaPlayerDevice, PLATFORM_SCHEMA, SUPPORT_NEXT_TRACK,
+    SUPPORT_PAUSE, SUPPORT_PLAY,
+    SUPPORT_PREVIOUS_TRACK, SUPPORT_STOP, SUPPORT_TURN_OFF,
+    SUPPORT_TURN_ON, SUPPORT_VOLUME_MUTE, SUPPORT_VOLUME_STEP)
+
+from homeassistant.const import (
+    ATTR_ENTITY_ID, CONF_DEVICES, CONF_NAME, CONF_PORT,
+    CONF_SCAN_INTERVAL, STATE_IDLE, STATE_PAUSED, STATE_PLAYING, STATE_OFF)
+from homeassistant.exceptions import PlatformNotReady
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.event import track_time_interval
+
+REQUIREMENTS = ['pure-python-adb>=0.1.5.dev0']
+
+_LOGGER = logging.getLogger(__name__)
+
+DEFAULT_NAME = 'Android'
+DEFAULT_PORT = 5555
+SCAN_INTERVAL = timedelta(seconds=10)
+
+SUPPORT_ANDROIDTV = (SUPPORT_NEXT_TRACK | SUPPORT_PAUSE |
+                     SUPPORT_PLAY | SUPPORT_PREVIOUS_TRACK |
+                     SUPPORT_TURN_OFF | SUPPORT_TURN_ON |
+                     SUPPORT_VOLUME_MUTE | SUPPORT_VOLUME_STEP |
+                     SUPPORT_STOP)
+
+
+DEVICE_SCHEMA = vol.Schema({
```

Comment:
Is there a reason to group multiple devices under one platform? Consider having a single device per platform config, which will make the config schema simpler. Multiple devices can still be set up via multiple platform configs.

Reply:
No there's not, I followed the Yeelight implementation and that's how it was done.
I'll switch to single device per platform.